mycroft/tts/__init__.py

```diff
@@ -341,11 +342,12 @@ def clear_cache(self):
         """ Remove all cached files. """
         if not os.path.exists(mycroft.util.get_cache_directory('tts')):
             return
-        for f in os.listdir(mycroft.util.get_cache_directory("tts")):
-            file_path = os.path.join(mycroft.util.get_cache_directory("tts"),
-                                     f)
-            if os.path.isfile(file_path):
-                os.unlink(file_path)
+        for d in os.listdir(mycroft.util.get_cache_directory("tts")):
+            dir_path = os.path.join(mycroft.util.get_cache_directory("tts"), d)
+            for f in os.listdir(dir_path):
```

Comment:
Should check if it's a file or directory I got error here when the cache directory was not empty of normal files. (as it will be after an update)